### DISCUSSION TITLE
Migrating from stand-alone to docker server

### DISCUSSION THREAD

**Discussion by Obliterous** [SaraSmiseth/rpi-prosody#68] [Q&A]
Am migrating from a long running stand-alone installation of prosody to this container, would like to move all of my users (new server will have same name/IP) what files do I need to move, and to where?

**Comment by SaraSmiseth** [comment]
If you use sqlite:
You need to copy your database file and mount it to the right place.

If you use the [example docker compose](https://github.com/SaraSmiseth/prosody?tab=readme-ov-file#run) file you can see that the folder `./data` is mounted to `/usr/local/var/lib/prosody`. The default config assumes your database file is called `prosody.sqlite`. So put it into `./data/prosody.sqlite`.

Depending on your old config you can adjust the [environment variables](https://github.com/SaraSmiseth/prosody?tab=readme-ov-file#environment-variables).
